catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -554,41 +528,41 @@ private Metacard copyMetacard(Metacard original) {
 
   private SolrDocumentList getSolrDocumentList(
       List<? extends Serializable> identifierPaged, String fieldName) throws IngestException {
+    // checking page size since real time get can only be done as a GET and not a POST
+    if (fieldName.equals(Metacard.ID + SchemaFields.TEXT_SUFFIX) && identifierPaged.size() <= 500) {
```

Comment:
So our id's (uuid v4, sans hyphens `-`) are 32 characters a piece - so that alone is 16k not accounting for anything else. 

Looks like the default in lucene-solr on the 7.4.0 tag caps it at 8k (if it is based off the jetty.xml config). Either way it seems like 500 could put that well over the limit as @dsmiley mentioned. Do we want to adjust this @pklinef or do we have another plan for handling this?